AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -16,6 +16,10 @@ turkishairlines.com,atmos-tokyo.com,radarbox.com,ciceksepeti.com,rabota.by,temp-
 ! notifications - part of the functionality
 ! discord.com##div[class^="container"] > nav[class^="wrapper"] + div[class^="base"] > div[class^="notice-"]
 !
+producthunt.com##[class*="styles_useAppButton"]
+producthunt.com##[class^="styles_wrapper"]
```

Comment:
To block the popup, better to use a scriptlet: 
`producthunt.com#%#//scriptlet('prevent-addEventListener', 'scroll', 'Math.round(e/n)!==r')`